go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	github.com/cloudevents/sdk-go v0.10.0
+	github.com/ghodss/yaml v1.0.0
```

Comment:
I'll check if we need this dependency, because this should work with the existing `yaml.v2` dependency...